### PR TITLE
Add feature to log output to a text file for persistance

### LINKED ISSUE #7: Save links to a file for future use
**Is your feature request related to a problem? Please describe.**
The results of the scraping are displayed on the terminal. A good to have feature would be to add the scraped profile links to a file if the user wants to look back or connect with someone else.

**Describe the solution you'd like**
- The output can be appended to a file with the name of the company inside a folder.
- Add the folder to .gitignore.

**Describe alternatives you've considered**
Printing on terminal is what we have for now

### PR DESCRIPTION
**Pull request by cwille97**
## Description

Adding a feature to log output from the script to a text file for persistance. This uses the built in Node 'fs' library. We were already using one method from this library. There script will now create an output directory if it isn't already there and then create text files based on the company name and a timestamp for uniqueness. 

Fixes #7 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: scrape.js
Comment (ayushjainrksh) [inline scrape.js:54]: Can we use a JSON file instead of `txt` file as it is easier to read and parse?

File: scrape.js
Comment (ayushjainrksh) [inline scrape.js:190]: It would be great if it can be updated to store data in the following format (JSON):

```
{
    "activeProfiles": {
        "page0": [
            "link1",
            "link2"
        ],
        "page1": [
            "link1",
            "link2"
        ]
    }
 }

```

**Review by ayushjainrksh — CHANGES_REQUESTED** [review]

**Comment by ayushjainrksh** [comment]
@cwille97 Great job with the PR! Works like a charm. Mind making a small change before I merge this? Everything else works fine.

**Comment by cwille97** [comment]
@ayushjainrksh Please review the changes for JSON, let me know what you think :)

File: scrape.js
Comment (ayushjainrksh) [inline scrape.js:190]: Sorry to block this PR again. There's a small bug here. When you run the script, you get `pageName` instead of `page0`, `page1`, etc. in the output file. This is because when you push `{pageName: currPage}` into a JSON, it uses `pageName` as a string instead of a variable. One way to solve this is to replace `pageName` with `[pageName]`.

**Review by ayushjainrksh — CHANGES_REQUESTED** [review]

**Comment by ayushjainrksh** [comment]
@cwille97 LGTM. Ready for merge after this change.

File: scrape.js
Comment (cwille97) [reply scrape.js:190]: Whoops, can't believe I glanced over that! Fixing now

Comment (cwille97) [reply scrape.js]: Should be all set now :) let me know if there's anything else!

**Review by ayushjainrksh — APPROVED** [review]

**Comment by cwille97** [comment]
Do I have to approve this also? It says that 1 person is still approving the review. After this gets merged I think we should be good to close the corresponding ticket.

**Comment by ayushjainrksh** [comment]
@cwille97 Sorry for the delay on PRs. I've already approved it and will merge it very soon. Thanks for your patience.

**Comment by cwille97** [comment]
@ayushjainrksh No problem! I didn't mean to rush you, I just wanted to make sure that nothing was missing that I needed to complete. Thanks! :)

**Comment by ayushjainrksh** [comment]
@cwille97 Thanks for the PR and sorry for the delay :)

**Pull request merged**
